apps/mcp/README.md

```
@@ -481,12 +538,183 @@
+   {
+     "jsonrpc": "2.0",
+     "id": 1,
+     "method": "tools/list",
+     "params": {}
+   }
+   ```
+
+#### Error Responses
+
+- **401 Unauthorized** - Missing or invalid token:
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)